axiom/tests/test_txn.c

```diff
@@ -8396,6 +8397,7 @@ static void test_txn_log_configuration(void) {
 
   /* log features globally disabled, high security disabled */
   txn->options.logging_enabled = false;
+  txn->options.log_decorating_enabled = false;
```

Comment:
Maybe consider adding one test for `log_decorating_enabled = true;`?

Reply:
Thanks for catching this - added several in this [commit](https://github.com/newrelic/newrelic-php-agent/commit/d11135c69cab321567726a62ba8ccead000e5746).